source/api/plugins/browser-launch-api.md

```diff
@@ -55,10 +57,57 @@ module.exports = (on, config) => {
     if (browser.name === 'electron') {
       // `args` is a `BrowserWindow` options object
       // https://electronjs.org/docs/api/browser-window#new-browserwindowoptions
-      args['fullscreen'] = true
+      options.preferences['fullscreen'] = true
+
+      // whatever you return here becomes the new launch options
+      return options
+    }
+
+    if (browser.family === 'firefox') {
+      options.firefoxOptions.windowSizeMode = 'fullscreen'
+
+      return options
+    }
+  })
+}
+```
+
+Add browser extensions:
+
+```js
+module.exports = (on, config) => {
+  on('browser:before:launch', (browser, options) => {
+    // supply the path to an unpacked WebExtension
+    // NOTE: extensions cannot be loaded in headless Chrome
+    options.extensions.push('/path/to/webextension')
+
+    return options
+  })
+}
+```
+
+Changing browser preferences:
+
+```js
+module.exports = (on, config) => {
+  on('browser:before:launch', (browser, options) => {
+    if (browser.family === 'chromium' && browser.name !== 'electron') {
```

Comment:
worth calling out that this targets all `chrome`, `chromium`, `canary`, `edge`, etc browser versions but excludes `electron`

Reply:
Well, this check is used in several places so I'm not sure we want to list out all browsers here necessarily.